README.md

```diff
@@ -1,2 +1,51 @@
 # ChapPress
-A simple project to model best practices for WordPress development.
+ChapPress is a simple WordPress project initiated by the Web and Interactive Marketing team for Chapman University. It is intended to to model best practices for WordPress development. This repository will host the Wordpress model site with custom child themes and plugins.
+
+## Goals
+This WordPress project is intended to provide:
+- A consistent local development environment
+- Source control using standard Git workflow practices
+- Modern automated tests
+- Documentation for best practices and standards
+
+## Requirements
+- Homebrew (1.2.3)
+  - MySQL (5.7.18)
+  - Wordpress Command Line (1.2.1)
+- PHP (5.6.30)
```

Comment:
For reference, our Ansible Mac workstation playbook installs PHP 7:

https://github.com/chapmanu/ansible-workstation